scripts/updateStaticMetadata.ts

```diff
@@ -0,0 +1,27 @@
+// Running this file with ~ node scripts/updateStaticMetadata.js will create a staticMetadata.ts on root directory which can used/copied
+// const Api = require('@cennznet/api').Api;
+// const fs = require('fs');
+// const staticMetadata = require('@cennznet/api/staticMetadata');
+// const Metadata  = require('@cennznet/types').Metadata;
```

Comment:
Can we remove the commented lines in this file?

Reply:
oh I did that.. in the following commit..

Reply:
ya, was done